src/many-ledger/tests/common/mod.rs

```diff
@@ -139,7 +138,7 @@ impl Setup {
         let public_key = PublicKey(id.public_key().to_vec().unwrap().into());
 
         let store_path = tempfile::tempdir().expect("Could not create a temporary dir.");
-        debug!("Store path: {:?}", store_path.path());
+        eprintln!("Store path: {:?}", store_path.path());
```

Comment:
Don't we disallow using `eprintln!`?

Reply:
The problem is we cannot pass in a trace subscriber. Also, clippy didn't complain :/

I'll add a allow above it with a comment.